pkg/kubectl/cmd/get/get_test.go

```diff
@@ -702,6 +703,38 @@ func TestGetObjectIgnoreNotFound(t *testing.T) {
 	}
 }
 
+func TestGetObjectClusterScopedNotFound(t *testing.T) {
+	cmdtesting.InitTestErrorHandler(t)
+	tf := cmdtesting.NewTestFactory()
+	defer tf.Cleanup()
+
+	codec := scheme.Codecs.LegacyCodec(scheme.Scheme.PrioritizedVersionsAllGroups()...)
+	tf.UnstructuredClient = &fake.RESTClient{
+		NegotiatedSerializer: resource.UnstructuredPlusDefaultContentConfig().NegotiatedSerializer,
+		Client: fake.CreateHTTPClient(func(req *http.Request) (*http.Response, error) {
+			return &http.Response{
+				StatusCode: 200, Header: cmdtesting.DefaultHeader(),
+				Body: cmdtesting.ObjBody(codec, &storagev1.VolumeAttachmentList{})}, nil
+		}),
+	}
+
+	streams, _, buf, errbuf := genericclioptions.NewTestIOStreams()
+	cmd := NewCmdGet("kubectl", tf, streams)
+	cmd.SetOutput(buf)
+	cmd.Run(cmd, []string{"volumeattachments"})
+
+	expected := ``
+	if e, a := expected, buf.String(); e != a {
+		t.Errorf("expected\n%v\ngot\n%v", e, a)
+	}
+	expectedErr := `No resources found
+`
```

Comment:
nit: quote is in the next line

Reply:
@hpandeycodeit it needs the new line to match the error string

Reply:
Do you mean the error string has a new line?

Reply:
@hpandeycodeit yes, i am just following the pattern in the rest of the file. like here https://github.com/kubernetes/kubernetes/pull/83384/files/678d57c341fb0dc68a9568e78cdd0a77a4dbc912#diff-6cc0ed84f5d654c3dc73c9a27f76226aL652-L653